Doc/library/zipfile.rst

```diff
@@ -456,6 +465,62 @@ The following data attributes are also available:
    truncated.
 
 
+.. _path-objects:
+
+Path Objects
+------------
+
+.. class:: Path(root, at='')
+
+   Construct a Path object from a root zipfile (which may be a
```

Comment:
This needs to be more explicit about the semantics and values for the `root` and `at` parameters.

Reply:
In c8c151113c, I added a section about the 'at' parameter. What do you think is missing about the `root` parameter?

Reply:
Maybe just mark up the first mention of `root` so that it's clear you're talking about the parameter?